sdk/monitor/OpenTelemetry.Exporter.AzureMonitor/src/AzureMonitorTransmitter.cs

```diff
@@ -60,6 +59,17 @@ internal async ValueTask<int> AddBatchActivityAsync(Batch<Activity> batchActivit
             {
                 telemetryItem = GeneratePartAEnvelope(activity);
                 telemetryItem.InstrumentationKey = this.instrumentationKey;
+
+                if (activity.Events.Count() > 0)
+                {
+                    foreach (var eventData in activity.Events)
+                    {
+                        var eventTelemetryItem = telemetryItem.Clone(PartA_Name_Mapping[TelemetryType.Event], eventData.Timestamp);
```

Comment:
Activity.Events should be mapped to TraceTelemetry.
`EventTelemetry` is for business events I believe, though not 100% certain.

Reply:
Probably worth a discussion/PR in the schema repo.